### PR TITLE
feat: enable egress tracking in production env

### PR DESCRIPTION
**Pull request by fforbeck**
Enabled Egress Tracking in the production environment, along with the KV Delegation Store for saving and retrieving `space/content/serve/*` delegations.

### CODE REVIEW

File: wrangler.toml [hunk 1]
```diff
@@ -51,13 +51,15 @@ command = "npm run build"
...
-FF_EGRESS_TRACKER_ENABLED = "false"
+FF_EGRESS_TRACKER_ENABLED = "true"
 FF_TELEMETRY_ENABLED = "true"
+FF_DELEGATIONS_STORAGE_ENABLED = "true"
 GATEWAY_SERVICE_DID = "did:web:w3s.link"
 UPLOAD_SERVICE_DID = "did:web:web3.storage"
 CONTENT_CLAIMS_SERVICE_URL = "https://claims.web3.storage"
 CARPARK_PUBLIC_BUCKET_URL = "https://carpark-prod-0.r2.w3s.link"
 UPLOAD_API_URL = "https://up.web3.storage"
+INDEXING_SERVICE_URL = "https://indexer.storacha.network/"
```
Comment (fforbeck) [inline wrangler.toml:62]: @alanshaw Could you confirm if this is the correct URL for the Production Indexer?

Comment (alanshaw) [reply wrangler.toml:62]: 👍

**Review by alanshaw — APPROVED** [review]
LGTM but please hold off pushing to prod until production indexing service is deployed.

**Comment by fforbeck** [comment]
> LGTM but please hold off pushing to prod until production indexing service is deployed.

sounds good. Thank you!

**Pull request merged**